CHANGELOG.md

```diff
@@ -19,7 +19,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
 ### Changed
--Fixed Python error when a class inherits from a base class and inmplents an interface.
+- Fixed Python error when a class inherits from a base class and implements an interface.
```

Comment:
this needs to be moved to the unreleased section. @musale can you take this on since shem it out please?

Reply:
Any chance you can double check this? I already pushed a commit to fix this one earlier....

Reply:
indeed, I was looking at an outdated commit. Thanks for pointing this out!